lox_services/persistence/database/push_invoices_data.py

```diff
@@ -58,16 +89,25 @@ def push_run_to_database(
             invoice_path,
             infer_datetime_format=date_format,
             header=0,
-            dtype={"postal_code_reciever": str},
+            dtype={"postal_code_reciever": "string[pyarrow]"},
         )
-        if df_invoice.shape[0] > 0:
-            # If some columns are missing, add the columns with None values
-            for column in dtypes_invoices:
-                if column not in df_invoice.columns:
-                    df_invoice.loc[:, column] = None
 
-            # "Make sure that the columns have the good type
-            df_invoice = df_invoice.fillna(value=na_invoices).astype(dtypes_invoices)
+        postal_code_cols = df_invoice.columns[
+            df_invoice.columns.isin({"postal_code_receiver", "postal_code_sender"})
+        ].tolist()
+        # In case postal_code_reciever falsely gets interpreted as a float
+        df_invoice[postal_code_cols] = (
```

Comment:
@VovaViliLox Can we have this in a small function?

Reply:
Both the remove of .0 and the validation of the country code cna be named

process_countries_column()

Reply:
If you prefer it this way - sure.